### PR TITLE
populate couch to sql command: add --domains flag and progress output

### PR DESCRIPTION
**Pull request by snopoke**
## Summary
The goal here was to make it easier to run the command for selected domains in order to give the user more scope for testing the command before final rollout. The progress output also makes it easier to see the progress and the log data is persisted to disk for reference so it doesn't get lost in the console.

* Adds a `--domains` option to the command to allow migrating only specific domains. This is useful for spot checking prior to running on all data.
* Write detailed log output to a file and only show progress info in the console.

Can be reviewed by commit.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Automated test coverage
None for this specifically though there are other tests for usages of this in specific migrations.

### QA Plan
Tested locally and used when running the command on `india`

### Safety story
This only affects management commands.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py [hunk 1]
```diff
@@ -238,3 +259,18 @@ def _migrate_doc(self, doc):
             model, created = self.update_or_create_sql_object(doc)
             action = "Creating" if created else "Updated"
             logger.info("{} model for doc with id {}".format(action, doc["_id"]))
+
+    def _get_couch_doc_count_for_domain(self, domain):
```
Comment (orangejenny) [inline corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py:275]: Nitpick: it'd be nice to have this expect a list so it's more identical to the sql version.

**Review by orangejenny — APPROVED** [review]
This is great

File: corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py (hunk 1 above)
Comment (snopoke) [reply corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py]: df5ee1060b8

**Review by mjriley — COMMENTED** [review]
Curious why logging was moved from the `logging` module to a file? It seems like keeping the `logging` module, where the module output both to the console and a file, would allow for greater flexibility (typically command-line applications like this allow increased logging with a 'verbose' flag, for example)

**Comment by snopoke** [comment]
The 'logging' here isn't really useful as console output because it's very voluminous (usually more than the max console history) and unless you're watching it all the time you likely won't see any issues that come up. It's also important to have it in a file for later reference. I could have changed the log levels so that only 'errors' were logged at a higher level but but with data migrations I think logging to a file is generally preferable and tracking all data that is touched.

**Pull request merged**